apps/web/src/app/[lang]/page.tsx

```diff
@@ -13,20 +14,23 @@ export default async function Index({ params: { lang } }: PageProps) {
   const daos = await getDaoList({ token: newtoken.accessJWT });
 
   return (
-    <div className="grid grid-cols-4 gap-4 px-6 py-4">
-      {daos.map((dao) => (
-        <div key={dao.name}>
-          <Link href={getAssignmentsPath(lang, dao.url as string)}>
-            <CardOrganisation
-              createdDate={dao.date}
-              description={dao.description as string}
-              icon={dao.logo}
-              members={0}
-              title={dao.title as string}
-            />
-          </Link>
+    <div className="px-6 py-4">
+      <Heading className="mb-4" size={9} color="#fff" weight="medium" align="center">All your spaces, in one place</Heading>
```

Comment:
There are some type errors for `color` and `weight`. See my comment on the component also